generators/server/index.js

```diff
@@ -148,6 +148,7 @@ module.exports = class extends BaseGenerator {
                 if (this.searchEngine === undefined) {
                     this.searchEngine = false;
                 }
+                this.jhiPrefix = this.configOptions.jhiPrefix || this.config.get('jhiPrefix') || this.options['jhi-prefix'];
```

Comment:
there is no jhi-prefix option on this sub gen hence this line should be
`this.jhiPrefix = this.configOptions.jhiPrefix || this.config.get('jhiPrefix');`

Reply:
10x for the hints